pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=75.6", "Cython==3.0.8", "numpy==1.26.3", "build", "cibuildwheel"]
+requires = ["setuptools>=75.6", "Cython==3.0.8", "numpy>=1.26.3,<2.0", "build", "cibuildwheel"]
```

Comment:
See comment above. I tested with a fresh install on a new M4 machine, and in spite of `<2.0`, I still got numpy 2.0.3 installed, so maybe we can keep the requirement of numpy to be 1.26.3?

Furthermore, we should also add scipy==1.14.0, given a recent issue with scipy 1.15. deprecating and removing `scipy.misc.derivative`, which is used in partitura (see [Isse 409](https://github.com/CPJKU/partitura/issues/409)). We should use scipy 1.14.0 until this issue is resolved

Reply:
https://github.com/pymatchmaker/matchmaker/pull/16/commits/509aba0aec8bb9877319d1c8b87983b32eaa4af7 Applied!